modules/util/path.go

```diff
@@ -182,6 +182,7 @@ func FileURLToPath(u *url.URL) (string, error) {
 // it returns error when the variable does not exist.
 func HomeDir() (home string, err error) {
 	// TODO: some users run Gitea with mismatched uid  and "HOME=xxx" (they set HOME=xxx by environment manually)
+	// TODO: when running gitea as a sub command inside git, the HOME directory is not the user's home directory
```

Comment:
This maybe a problem for git hooks sub command?

Reply:
I think there is no problem, it's just a tip for future developers who misuse the HOME (indeed, in most cases, the HOME from environment shouldn't be used, it can not be stable)